elasticdownload/src/main/java/is/arontibo/library/ElasticDownloadView.java

```diff
@@ -107,5 +107,30 @@ public void onEnterAnimationFinished() {
 
         // Do further actions if necessary
     }
+    
+
+    /**
+     * Set the background color of the Elastic Download View
+     * @param passedColor int Color Color to set the background
+     */
+    public void setBackgroundColor(int passedColor) {
+        this.mBackgroundColor = passedColor;
+        this.mProgressDownloadView.setBackgroundColor(mBackgroundColor);
+    }
+
+    /**
+     * Overloaded method, takes in a String color to parse
+     * @param passedColor String of a color hex color (IE: #fd5c79) that can be parsed by
+     *                    Color.parseColor(string)
+     */
+    public void setBackgroundColor(String passedColor) {
+        if(passedColor == null){
+            return;
+        }
+        try {
+            int color = Color.parseColor(passedColor);
+            this.setBackgroundColor(color);
+        } catch (IllegalArgumentException e) {}
```

Comment:
Better! What we usually do in the case is use `e.printstacktrace();` to notify in logcat that something when wrong, or you can use a log message with `Log.e`. Either way we should do something here, so developers can get notified when a wrong color has been passed to this function.